fireant/slicer/widgets/reacttable.py

```diff
@@ -69,25 +69,20 @@ def fillna_index(index, value):
     return index.fillna(value)
 
 
-def _get_item_display(item, value):
-    if item is None or all([item.prefix is None, item.suffix is None, item.precision is None]):
-        return
-
-    return '{prefix}{value}{suffix}'.format(
-          prefix=(item.prefix or ""),
-          suffix=(item.suffix or ""),
-          value=value if item.precision is None else '{:.{precision}f}'.format(0.1234567890, precision=2)
-    )
+def display_value(value, prefix=None, suffix=None, precision=None):
+    if isinstance(value, bool):
+        value = str(value).lower()
 
+    formatted_value = ('{:.{precision}f}'.format(value, precision=precision)
+                       if precision is not None and isinstance(value, float) else
+                       '{:,}'.format(value)
```

Comment:
Is it possible to split this? This is really hard to follow.

Reply:
would be nice if python supported switch statements. Does it help to follow it if you think of it as `condition 1 -> outcome 1 ELSE condition 2 -> outcome 2 ELSE outcome 3`?

Reply:
i broke it up to make it easier to read

Reply:
I personally think it would be a lot easier to read just with separated if statements.